### PR TITLE
Initial Devise implementation

### PR DESCRIPTION
**Pull request by csutter**
This adds a trivial Devise model to the app, complete with some basic
views that aren't wired up to anything yet. It also takes out all the
fluff from the Devise generator to leave just the stuff that we need.

### CODE REVIEW

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  # to behave the same regardless if the e-mail provided was right or wrong.
+  # Does not affect registerable.
+  # config.paranoid = true
+
+  # By default Devise will store the user in session. You can skip storage for
+  # particular strategies by setting this option.
+  # Notice that if you are skipping storage for all authentication paths, you
+  # may want to disable generating routes to Devise's sessions controller by
+  # passing skip: :sessions to `devise_for` in your config/routes.rb
+  config.skip_session_storage = []
```
Comment (digitalronin) [inline config/initializers/devise.rb]: Is the intention here that we don't store the user in the session, or that we don't skip storing the user in the session!? Not really a comment on the code so much as a comment on the naming & setting of the config value.

Comment (csutter) [inline config/initializers/devise.rb]: AFAIK the point of this config is for things like token auth where you want each request to authenticate again. We _don't_ want to _skip_ (loving the double negative) anything so we leave the array empty.

Comment (digitalronin) [inline config/initializers/devise.rb]: Awesome. Thanks for that. Yay for naming things.

File: config/locales/devise.en.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
+en:
```
Comment (digitalronin) [inline config/locales/devise.en.yml]: Please could you merge this file into config/locales/en.yml? 

Now that we've got everything into one yaml file, we should keep that consistent (it makes Josh's life easier, too). Everything is under en:devise, so it should be pretty trivial to do.

Comment (csutter) [inline config/locales/devise.en.yml]: Sure thing – will do.

**Pull request merged**